### ISSUE TITLE
Error Loading KEGG Glycolysis

### ISSUE THREAD

**Issue by cannin** [PathwayCommons/pathways-search#15]
Glycolysis from KEGG did not load on OSX Chrome/Safari or Android Chrome. Messages in console: 

    bundle.js:6 Warning: It looks like you're using a minified copy of the
    development build of React. When deploying React apps to production,
    make sure to use the production build which skips development warnings
    and is faster. See https://fb.me/react-minification for more details.
    e @ bundle.js:6
    bundle.js:1 RENDER START

    http://www.pathwaycommons.org/pc2/get?format=SBGN&user=pc2pathways&uri=http%3A%2F%2Fidentifiers.org%2Fkegg.pathway%2Fhsa00010
    Failed to load resource: the server responded with a status of 502
    (Proxy Error)
    pathways-search.html#/pathway?name=Glycolysis+%2F+Gluconeogenesis&uri=http%3A%2F%2Fidentifiers.org%2Fkegg.pathway%2Fhsa00010:1
    XMLHttpRequest cannot load
    http://www.pathwaycommons.org/pc2/get?format=SBGN&user=pc2pathways&uri=http%3A%2F%2Fidentifiers.org%2Fkegg.pathway%2Fhsa00010.
    No 'Access-Control-Allow-Origin' header is present on the requested
    resource. Origin 'http://mj3cheun.github.io' is therefore not allowed
    access. The response had HTTP status code 502.

**Comment by mj3cheun** [comment]
Hi Augustin, about the error you got, based on the symptoms you reported you probably tried to load a pathway that is too large. What happens is the page requests the SBGNML resource from the pathways server and tries to download it but never finishes due to a 5XX error which indicates the failure is server side. The largest pathways I have tested successfully are listed on the search app as size 500 - 600, and even then it took roughly 5 - 10 minutes to download the SBGNML file and render.

Addendum: I have tried loading the following link (http://www.pathwaycommons.org/pc2/get?format=SBGN&user=pc2pathways&uri=http%3A%2F%2Fidentifiers.org%2Fkegg.pathway%2Fhsa00010) just in the browser with the same result, the server just sends a 502 back after a while. So confirmed this is an issue with pathwaycommons itself. However, we can investigate better error handling both server and client side.
